src/web/layouts/InteractiveLayout.tsx

```diff
@@ -0,0 +1,622 @@
+				margin-left: 0px;
+			}
+			${from.leftCol} {
+				margin-left: 151px;
+			}
+			${from.wide} {
+				margin-left: 230px;
+			}
+
+			@supports (display: grid) {
+				display: grid;
+				width: 100%;
+				margin-left: 0;
+
+				grid-column-gap: 10px;
+
+				${from.wide} {
+					grid-template-columns:
+						219px /* Left Column (220 - 1px border) */
+						1px /* Vertical grey border */
+						1fr; /* Main content */
+
+					grid-template-areas:
+						'title  border  headline'
+						'.      border  standfirst'
+						'lines  border  media'
+						'meta   border  media'
+						'meta   border  body'
+						'.      border  .';
+				}
+
+				${until.wide} {
+					grid-template-columns:
+						140px /* Left Column */
+						1px /* Vertical grey border */
+						1fr; /* Main content */
+
+					grid-template-areas:
+						'title  border  headline'
+						'.      border  standfirst'
+						'lines  border  media'
+						'meta   border  media'
+						'meta   border  body'
+						'.      border  .';
+				}
+
+				${until.leftCol} {
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'title'
+						'headline'
+						'standfirst'
+						'media'
+						'lines'
+						'meta'
+						'body'
+						'.';
+				}
+
+				${until.desktop} {
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'title'
+						'headline'
+						'standfirst'
+						'media'
+						'lines'
+						'meta'
+						'body';
+				}
+
+				${until.tablet} {
+					grid-column-gap: 0px;
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'media'
+						'title'
+						'headline'
+						'standfirst'
+						'lines'
+						'meta'
+						'body';
+				}
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const maxWidth = css`
+	${from.desktop} {
+		max-width: 620px;
+	}
+`;
+
+const articleWidth = css`
+	${from.desktop} {
+		width: 620px;
+	}
+`;
+
+const stretchLines = css`
+	${until.phablet} {
+		margin-left: -20px;
+		margin-right: -20px;
+	}
+	${until.mobileLandscape} {
+		margin-left: -10px;
+		margin-right: -10px;
+	}
+`;
+
+const starWrapper = css`
+	margin-bottom: 18px;
+	margin-top: 6px;
+	background-color: ${brandAltBackground.primary};
+	display: inline-block;
+
+	${until.phablet} {
+		padding-left: 20px;
+		margin-left: -20px;
+	}
+	${until.leftCol} {
+		padding-left: 0px;
+		margin-left: -0px;
+	}
+
+	padding-left: 10px;
+	margin-left: -10px;
+`;
+
+const ageWarningMargins = css`
+	margin-top: 12px;
+	margin-left: -10px;
+	margin-bottom: 6px;
+
+	${from.tablet} {
+		margin-left: -20px;
+	}
+
+	${from.leftCol} {
+		margin-left: -10px;
+		margin-top: 0;
+	}
+`;
+
+interface Props {
+	CAPI: CAPIType;
+	NAV: NavType;
+	format: Format;
+	palette: Palette;
+}
+
+export const InteractiveLayout = ({ CAPI, NAV, format, palette }: Props) => {
+	const {
+		config: { isPaidContent, host },
+	} = CAPI;
+
+	const adTargeting: AdTargeting = buildAdTargeting(CAPI.config);
+
+	const showBodyEndSlot =
```

Comment:
Is the epic on interactives? Can we kill this?

Reply:
Good point, I don't think so. I'll remove for now and we can always re-add if needed. (Though I suspect this code will change a lot anyway.)